tests/compute/subresources/rbac.go

```diff
@@ -47,25 +48,25 @@ var _ = compute.SIGDescribe("[rfe_id:1195][crit:medium][vendor:cnv-qe@redhat.com
 			saClient = getClientForSA(kubevirt.Client(), testsuite.SubresourceServiceAccountName)
 		})
 
-		It("[test_id:3170]should allow access to vm subresource endpoint", func() {
+		It("[test_id:3170]should allow access to vm subresource endpoint", decorators.Conformance, func() {
```

Comment:
Similar as to the rbac tests, I am not sure this is good candidate

Reply:
@fossedihelm You seem to agree but I see that this is still included. What do you plan to do with this?

Reply:
@xpivarc Sorry, I probably missed that during the rebase.
Dropped the commit.